run_app_interactive_mode.py

```diff
@@ -33,19 +33,27 @@ def click_event(event, x, y, _flags, _params):
         """Sets action when cv2 mouse-callback is detected, i.e. user has clicked."""
         if event == cv2.EVENT_LBUTTONDOWN:
             current_action = np.array([x, y])
-            ob, rew, _, info = app.step(rescale_values(current_action,
-                                                       previous_low=0,
-                                                       previous_high=app.get_screen_size(),
-                                                       new_low=-1,
-                                                       new_high=1))
+
+            # If the screen is square, we only need one value to specify the action
+            # Otherwise, we need two values
+            if (app.get_screen_height == app.get_screen_width):
+                ob, rew, _, info = app.step(rescale_values(current_action,
+                                                           previous_low=0,
+                                                           previous_high=app.get_screen_size(),
+                                                           new_low=-1,
+                                                           new_high=1))
+            else:
+                ob, rew, _, info = app.step([rescale_values(current_action[0], previous_low=0, previous_high=app.get_screen_width(), new_high=1, new_low=-1),
+                                             rescale_values(current_action[1], previous_low=0, previous_high=app.get_screen_height(), new_high=1, new_low=-1)])
```

Comment:
Könnte man vereinfachen indem man einfach nur das vom else Block verwendet -> Das ist ja in beiden Fällen richtig